M365/MDO/MDOThreatPolicyChecker.ps1

```diff
@@ -918,5 +1009,8 @@ process {
             }
         }
     }
+    if (-not $ShowDetailedExplanation) {
+        Write-Host ("`nFor details about why a policy applies to a recipient, use the -ShowDetailedExplanation parameter and run this script again.")
+    }
     Write-Host " "
-}
+}
```

Comment:
It looks it doesnt ends with a empty end line:
![image](https://github.com/user-attachments/assets/b3458003-3694-472b-8734-631b288aa067)